README.md

```diff
@@ -10,7 +10,7 @@ a new CDN.
 Not familiar with Font Awesome 5? [Learn
 more](https://www.kickstarter.com/projects/232193852/font-awesome-5) about our
 successful Kickstarter and plan. You can also **[order Font Awesome
-Pro](https://fontawesome.com/pro)** which includes tons more icons directly
+Pro](https://fontawesome.com/pro)** which includes tons of  icons directly
```

Comment:
I think this means that FA Pro has more icons than FA Free.

Does changing `more` to `of` give the same idea?

Reply:
@Anu082000 this changes the meaning. Was there something confusing about the original sentence?